fastlane/spec/cli_tools_distributor_spec.rb

```diff
@@ -33,6 +33,11 @@
   end
 
   describe "update checking" do
+    before(:each) do
+      # prevent the update checker to run and clutter the output
+      ENV["FASTLANE_SKIP_UPDATE_CHECK"] = "a_value"
```

Comment:
```suggestion
      ENV["FASTLANE_SKIP_UPDATE_CHECK"] = "true"
```

Reply:
this was on purpose, to show that the value really doesn't matter. WDYT?

Reply:
Ah, gotcha. I didn't realize that the test and spec you introduced that change on was the one relevant for testing this.

But then I'm confused: that means that before your change, the skip env var was not set, so I'm guessing that means that all the test examples under `describe "update checking"` were actually running the fastlane update check… but if you set the env var value now, won't that disable the fastlane update check that those tests were supposed to run in order to test the `update checking` family of tests?

Or are you saying that a value of `"a_value"` is supposed to be the same as `"0"` or `"false"`, not `"true"`?

I haven't checked how the value of this env var is checked in the implementation to be honest; I guess it depends if it's checked with our `is_truthy?` helper—which only consider values like `"1"`, `"YES"`, `"true"`… as true—or if it's just checked like `ENV["…"].nil? || ENV["…"].empty?`—in which case any non-empty value would be considered as `true`, including `"a_value"` 🤔 💭

Reply:
Ok as soon as I wrote this comment I did a quick grep and saw that `FASTLANE_SKIP_UPDATE_CHECK` was always checked using `FastlaneCore::Env.truthy?` after all. (except in `spec_helper.rb:6`… but that's a test anyway)

So that means that `"a_value"` would be equivalent to `false`, not `true` as I initially thought.

Better make the intention more clear with a proper dummy value still:

```suggestion
      ENV["FASTLANE_SKIP_UPDATE_CHECK"] = "a_falsy_value"
```